staging/src/k8s.io/apiextensions-apiserver/test/integration/ratcheting_test.go

```diff
@@ -1295,13 +1303,54 @@ func TestRatchetingFunctionality(t *testing.T) {
 		},
 		{
 			Name: "CEL_transition_rules_should_not_ratchet",
+			Operations: []ratchetingTestOperation{
+				updateMyCRDV1Beta1Schema{&apiextensionsv1.JSONSchemaProps{
+					Type:                   "object",
+					XPreserveUnknownFields: ptr(true),
+				}},
+				applyPatchOperation{
+					"create instance with strings that do not start with k8s",
+					myCRDV1Beta1, myCRDInstanceName,
+					`
+                        myStringField: myStringValue
```

Comment:
OK I think the problem here is that you indent these with spaces rather than tabs, and then they look different in your environment than they do in github, it's really confusing me a lot for some reason.

Reply:
I have to use spaces or they dont parse as yaml :)

Reply:
![image](https://github.com/kubernetes/kubernetes/assets/1929644/a618329f-4f95-411c-9f24-34ef25e3faa8)

Reply:
<img width="578" alt="Screenshot 2023-10-17 at 5 46 19 PM" src="https://github.com/kubernetes/kubernetes/assets/351783/d7634db8-eedb-42a7-a5aa-59850be66344">

change ur tab to render with 4 spaces itll look right

Reply:
added FIxTabsOrDie